### DISCUSSION TITLE
upload_to_testflight weak documentation related to usage of app specific password

### DISCUSSION THREAD

**Discussion by peterlauri** [fastlane/fastlane#17658] [General]
https://docs.fastlane.tools/actions/upload_to_testflight/#use-an-application-specific-password-to-upload

This documentation states that one can use app specific passwords if **skip_waiting_for_build_processing** and **apple_id** is set. This is correct. But it might take a novice user to fully understand that **apple_id** is not the username (as in the Appfile). It is actually the App's Apple ID, not the user. This is really confusing.

Sure, it is stated in the documentation for **apple_id** in the upload_to_testflight parameters documentation. But it is not clear enough. The parameter should be named **app_apple_id** to not confuse the user with a normal Apple ID.

**Comment by max-ott** [comment]
Please feel free to open a pull request for the docs and changing this / commenting on that. I don't think a huge discussion is needed for that.

**Comment by peterlauri** [comment]
I tried to make a PR, but didn't really find the documentation. If you can
direct me where the documentation is I would be happy to make a PR :)

**Reply by max-ott** [reply]
Go to https://docs.fastlane.tools/actions/upload_to_testflight/#use-an-application-specific-password-to-upload > Scroll to the top > Click on "Edit on Github" at the top right of the page (blue link)

**Comment by peterlauri** [comment]
Unfortunately, it is not that easy. Click on the "Edit on Github" and it
will not be straightforward, as it is module documentation, not just plain
documentation. I need to find where that part is documented in the repo etc.

**Reply by max-ott** [reply]
https://github.com/fastlane/docs/blob/master/docs/actions/upload_to_testflight.md
